Getting-Started/Setup/Server-Setup/azure-web-apps.md

```diff
@@ -45,11 +45,35 @@ ability of Azure Web Apps then you need to consult the
 [Load Balancing documentation](load-balancing.md)__ since there is a lot more that needs
 to be configured to support scaling/auto-scaling.
 
-* If you have a {machinename} token in your ~/Config/ExamineIndex.config file remove this part of the path. Example, if you have path that looks like: "~/App_Data/TEMP/ExamineIndexes/{machinename}/External/" it should be "~/App_Data/TEMP/ExamineIndexes/External/" 
-* Due to the nature of Lucene files and IO latency, you should update all of your Indexers and Searchers in the ~/Config/ExamineSettings.config file to have these two properties (see [here](http://issues.umbraco.org/issue/U4-7614) for more details):
-```
-useTempStorage="Sync"
-```
 * You should ensure that `fcnMode="Single"` in your web.config's `<httpRuntime>` section (this is the default that is shipped with Umbraco, see [here](http://shazwazza.com/post/all-about-aspnet-file-change-notification-fcn/) for more details)
-* You should set your log4net minimum log priority to "WARN" in /Config/log4net.config if you are running a live site (of course if you are debugging this is irrelavent)
-* The minimum recommended Azure SQL Tier is "S2", however noticable performance improvements are seen in higher Tiers 
\ No newline at end of file
+* You should set your log4net minimum log priority to "WARN" in /Config/log4net.config if you are running a live site (of course if you are debugging this is irrelevant)
+* The minimum recommended Azure SQL Tier is "S2", however noticeable performance improvements are seen in higher Tiers 
+
+#### Examine v0.1.80+ ####
+
+Examine v0.1.80 introduced a new `directoryFactory` which should be added to all indexers in the `~/Config/ExamineSettings.config` file
+
+    directoryFactory="Examine.LuceneEngine.Directories.SyncTempEnvDirectoryFactory,Examine"
+
+The `SyncTempEnvDirectoryFactory` enables Examine to sync indexes between the remote file system and the local environment temporary storage directory, the indexes will be accessed from the temporary storage directory.
```

Comment:
I think we should also use the copy about IO Latency here to make it clear about why this is important and what it's actually doing